libcxx/include/__algorithm/ranges_stable_sort.h

```diff
@@ -45,7 +47,18 @@ struct __stable_sort {
     auto __last_iter = ranges::next(__first, __last);
 
     auto&& __projected_comp = std::__make_projected(__comp, __proj);
-    std::__stable_sort_impl<_RangeAlgPolicy>(std::move(__first), __last_iter, __projected_comp);
+    constexpr auto __default_comp        = is_same_v<_Comp, ranges::less>;
+    constexpr auto __default_proj        = is_same_v<_Proj, identity>;
```

Comment:
These should be `__desugars_to_v` and `__is_identity` instead. That also removes the need to do anything in `ranges::stable_sort`.

Reply:
Done.